frontend/src/components/SearchBar/FindMorePropertiesForm.tsx

```diff
@@ -112,7 +112,7 @@ const FindMorePropertiesForm = <T extends any>(props: any) => {
             label="Construction type"
           />
           <VerticalLine />
-          <NumberInput label="Net rentable area" field="rentableArea" />
+          <NumberInput label="Net usable area" field="rentableArea" />
```

Comment:
The other labels use caps.  Any reason to keep this one small?

Reply:
Seems that's how Bobbi wants it, I just checked the UX pin